packages/test-devtools-evm-foundry/contracts/TestHelperOz5.sol

```diff
@@ -50,6 +50,29 @@ contract TestHelperOz5 is Test, OptionsHelper {
         SimpleMessageLib
     }
 
+    struct EndpointSetup {
+        EndpointV2[] endpointList;
+        uint32[] eidList;
+        address[] sendLibs;
+        address[] receiveLibs;
+        address[] signers;
+        PriceFeed priceFeed;
+    }
+
+    struct LibrarySetup {
+        SendUln302 sendUln;
+        ReceiveUln302 receiveUln;
+        Executor executor;
+        DVN dvn;
+        ExecutorFeeLib executorLib;
+        DVNFeeLib dvnLib;
+    }
+
+    struct ConfigParams {
+        IExecutor.DstConfigParam[] dstConfigParams;
```

Comment:
Sorry, another nit:  can we call this `executorConfigParams` to be symmetrical with `dvnConfigParams` below?

Reply:
I did this in a follow up.